configs/pcrf-no-mongodb/pcrf-default.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
TBH it looks a bit weird that being all the config files yaml this is done in JSON. I forsee this will be NACKed upon review. If I were you I'd move this to yaml, drop cJSON and even incorporate this as a node in pcrf.cfg